cns/service/main.go

```diff
@@ -562,6 +567,18 @@ func main() {
 			return
 		}
 
+		logger.Printf("Starting SyncHostNCVersion")
```

Comment:
Can you move this logic to a different file, Follow how IPAMPoolMonitor is initialized

Reply:
Not sure what's the benefit of moving to a different file. Keep it here we can also leverage the advantage of using context with timeout and retry every second.